.ci/scripts/lint-commits.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
+set -xeo pipefail
 
-set -xe
+export PATH=${PATH}:/$(pwd)/node_modules:/$(pwd)/node_modules/.bin
```

Comment:
This technically shouldn't be a problem, but why add `./node_modules` to the `PATH`? If there's no need for it it's less confusing if it's removed so we only have `./node_modules/.bin`. Though normally I'm in favor of just running the command with the full path instead of modifying `PATH`. But in this case, it's pretty self-contained, so I guess it doesn't have any weird side-effects 😅

Reply:
it is only for CI use

Reply:
What's the reason you have `/$(pwd)/node_modules` in there as well besides `/$(pwd)/node_modules/.bin`?

Reply:
those are the folders where  Node.js put binaries, that's it.

Reply:
There's binaries in `/$(pwd)/node_modules/.bin`, but there shouldn't be anything directly in `/$(pwd)/node_modules`, so we can remove that.